build/makelib/helm.mk

```diff
@@ -43,6 +43,9 @@ $(HELM_OUTPUT_DIR)/$(1)-$(VERSION).tgz: $(HELM) $(HELM_OUTPUT_DIR) $(shell find
 	@echo === helm package $(1)
 	@cp -r $(HELM_CHARTS_DIR)/$(1) $(OUTPUT_DIR)
 	@$(SED_CMD) 's|VERSION|$(VERSION)|g' $(OUTPUT_DIR)/$(1)/values.yaml
+	@$(SED_CMD) 's|# clusterNamespaces:|clusterNamesspaces:|g' $(OUTPUT_DIR)/$(1)/values.yaml
```

Comment:
What are the makefile changes for? Local testing? We wouldn't want to enable cluster-ns1 and cluster-ns2 in the build.

Reply:
>  Local testing?

Yes, the change was for local testing

> We wouldn't want to enable cluster-ns1 and cluster-ns2 in the build.

Okay I understood. Then, I will modify helm tests for passing values for `--set` option which would override values when installing chart.

`--set` option: https://v2.helm.sh/docs/helm/#helm-install